index.html

```diff
@@ -47,6 +47,212 @@ <h1>I’m Samuel Glad to see you!</h1>
+                            </li>
+                        </div>
+                        <div class="js">
+                            <li>
+                                <a href="#">javascript</a>
+                            </li>
+                        </div>
+                        
+                    </ul>
+                </div>
+                <div class="action">
+                    <div class="button">
+                        <p>See Project</p>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
+    <section class="about_myself">
+            <div class="pr-t">
+                <h1>About Myself</h1>
+            </div>
+            <div class="sec-t">
+                <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            </div>
+            <div class="soci-med">
+                <p>LET’S CONNECT</p>
+            </div>
+            <div class="socm-icons">
+                <ul class="icons">
+                    <li>
+                        <a href="#"><img src="images/Vector.svg" alt="twitter icons"></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="images/linkedn.svg" alt="linkedn icons"></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="images/medium.svg" alt="medium icons"></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="images/github.svg" alt="github icons"></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="images/finger.svg" alt="finger icons"></a>
+                    </li>
+                </ul>
+            </div>
+            <div class="normalbutton">
+                <p>Get my resume</p>
+            </div>
+            <div class="skill">
+                <div class="frame1">
+                    <div class="frame7">
+                        <p>Languages</p>
+                        <img src="images/Union.svg" alt="drop down icon">
+                    </div>
+                    <div class="l-list">
+                        <img src="images/Frame 9.png" alt="javascript icons">
+                        <img src="images/Frame 10.png" alt="css icons">
+                        <img src="images/Frame 11.png" alt="html icon">
+                    </div>
+                </div>
+                <div class="frame2">
+                    <p>Frameworks</p>
+                    <img src="images/ri.png" alt="right shift icon">
+                </div>
+                <hr class="hl">
+                <div class="frame3">
+                    <p>Skills</p>
+                    <img src="images/ri.png" alt="right shift icon">
+                </div>
+                <hr class="hl">
```

Comment:
- [ ] Kindly layout your `Language`, `Frameworks` and `Skills` in a `ul` and `li` format. Unordered list are used to group similar elements together by convention. This also ease your styling.
- [ ] Also, instead of using image to represent your lists, a best practice here is to use you normal `li` for each of those items. Kindly remove the images and write the contents i.e. `html`, `css`, javascript`, Ruby on rails` etc.. in a `li` relationship with a parent (`ul`) . I hope this makes sense :thinking: